riotfile.py

```diff
@@ -103,6 +103,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         "DD_CIVISIBILITY_AGENTLESS_ENABLED": "1",
         "DD_CIVISIBILITY_CODE_COVERAGE_ENABLED": "1",
         "DD_CIVISIBILITY_ITR_ENABLED": "1",
+        "DD_PATCH_MODULES": "unittest:false",
```

Comment:
Why do we need this? Can we add comment to describe how enabling the unittest integration impacts ddtrace tests.

Reply:
Some integrations use `unittest` TestCases which, since it is patched, ends up generating more spans. This is a problem with our `pytest` integration too, since it supports `unittest` as shown [here](https://docs.pytest.org/en/7.1.x/how-to/unittest.html).